source/operating-a-service/incident-playbook.html.md

```diff
@@ -0,0 +1,91 @@
+4.  Invite the appropriate people from the contact lists to the incident channel.
+
+### 4. Provide a service update to users outside DfE (comms lead)
+
+The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). During an incident, the comms lead needs to explain what’s happening to users outside DfE. The comms lead will need a GitHub account to do this, or delegate updates to a colleague who has one.
+
+The updates are managed via GitHub Actions and issues on the [teacher-services-upptime repository on GitHub](https://github.com/DFE-Digital/teacher-services-upptime). If a service’s automatic health check is failing continuously, an issue will be created within 5 minutes of the failure occurring and the dashboard will start reporting a service issue.
+
+To update the dashboard:
+
+1. Navigate to the appropriate incident issue on the [GitHub issues page](https://github.com/DFE-Digital/teacher-services-upptime/issues)
+2. Add a comment to the issue
+
+### 5. Start the incident report (any incident lead)
+
+Create the incident report using the template in Google Drive:
+
+- Create a running [Incident Report using this template](https://docs.google.com/document/d/1HwKCPafnluOIhIAWbSD91zxt7w3q4FGDIVKS3d_SDFA/edit?usp=sharing)
+- Rename the created file to include today’s date and save as a new file in the [Incident reports folder](https://drive.google.com/drive/folders/12uWIF4beypUpEjejTRcKtV2PFadT5met)
+
+### 6. Decide whether to contact users about an incident (support lead)
+
+Contact your users if:
+
+* The incident will negatively impact them for a prolonged period of time
+* It can pre-empt a high volume of support tickets
+
+Informing users about incidents is generally considered best practice, but should be decided on a case by case basis with the product and service managers.
+
+## While the incident is in progress (all incident leads)
+
+Keep all conversations and status updates about the incident on the dedicated Slack incident channel.
+
+Use these incident stages in your Slack updates:
+
+- Incident has occurred
+- Incident is being assessed
+- Incident is being fixed
+- Incident is resolved
+
+Use the Slack IncidentBot `/update` command to update:
+
+- Description
+- Priority
+- Leads
+
+### Provide regular updates every 60 minutes (comms lead)
+
+Update stakeholders on the Slack incident channel every 60 minutes, until the incident has been resolved. Ensure they receive the alert even if their Slack alert notifications are turned off, and check in with them face-to-face (once back in the office).
+
+## Once the incident is resolved
+
+1. Update the running incident report
+2. Close the incident on using `/closeincident` command in Slack
+3. Confirm that the incident has been automatically resolved on the service status dashboard (it may take 5 mins to update)
```

Comment:
```suggestion
3. Confirm that the incident has been automatically resolved on the [service status dashboard](https://teacher-services-status.education.gov.uk/) (it may take 5 mins to update)
```